### PR TITLE
Addiitonal methods in event handling

### PR DESCRIPTION
**Pull request by hendrikebbers**
Some additional methods to subscribe / unsubscribe for specific event types.
added some default values for priority.

### CODE REVIEW

File: api/src/main/java/javax/application/event/EventHandler.java [hunk 1]
```diff
@@ -30,6 +30,22 @@
...
+
+    /**
+     * Default priority
+     */
+    int DEFAULT_PRIORITY = 0;
+
+    /**
+     * Default value for a high priority.
+     */
+    int HIGH_PRIORITY = 100;
```
Comment (aalmiray) [inline api/src/main/java/javax/application/event/EventHandler.java]: why restrict these values to `-100` and `100`? I'd leave open to interpretation for each implementor to decide the number range, same as it is with `Comparable` and implementations of `compareTo(O)`

Comment (hendrikebbers) [reply api/src/main/java/javax/application/event/EventHandler.java]: I was thinking just as a hint for them. But yes, maybe we should start just with the default value. Will change it

**Comment by hendrikebbers** [comment]
done

**Comment by aalmiray** [comment]
These methods do not belong to the base `EventBus` type. They might be provided by a particular implementor perhaps if deemed useful.

**Pull request closed without merging**